docs/user/03-10-preconfigured-secret.md

```diff
@@ -47,10 +47,13 @@ By default, SAP BTP Operator has cluster-wide permissions.
 
 The following parameters manage cluster access:
 
-| Parameter                     | Description                                                                                   |
-|-------------------------------|-----------------------------------------------------------------------------------------------|
-| **CLUSTER_ID**                | Generated when Kyma runtime is created.                                                       |
-| **MANAGEMENT_NAMESPACE**      | By default, set to `kyma-system`.                                                |
-| **ALLOW_CLUSTER_ACCESS**      | You can use every namespace for your operations. The parameter is always set to `true`.<br>If you change it to `false`, the setting is automatically reverted. |
+| Parameter                | Description                                                                                                                                                                              |
+|--------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **CLUSTER_ID**           | Generated when Kyma runtime is created.                                                                                                                                                  |
+| **MANAGEMENT_NAMESPACE** | Indicates the namespace for Secrets referenced in service instances and Secrets with the name containing a prefix of the service instance's namespace. By default, set to `kyma-system`. |
```

Comment:
```suggestion
| **MANAGEMENT_NAMESPACE** | Indicates the namespace for Secrets referenced in service instances and Secrets, with the name containing a prefix of the service instance's namespace. By default, set to `kyma-system`. |
```
I'm not sure the comma is grammatically necessary, but considering the sentence is pretty long, I think it might improve comprehension.